meta/main.yml

```diff
@@ -20,12 +20,8 @@ galaxy_info:
   platforms:
     - name: Debian
       versions:
-        - stretch
         - buster
         - bullseye
-        # Kali linux isn't an option here, but it is based on
-        # Debian Testing:
-        # https://www.kali.org/docs/policy/kali-linux-relationship-with-debian
         - bookworm
     - name: Ubuntu
       versions:
```

Comment:
We test against Kali Linux in the Molecule tests, so it should be included here as a supported platform.

Reply:
Nobody wants you and your logic!  You can see 02d686c anyway